Eslint not working and doesn't show errors

### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

I don't see any eslint diagnostics issues.

### Environment

Zed: v0.121.7 (Zed)
OS: macOS 14.0.0
Memory: 16 GiB
Architecture: aarch64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

_No response_

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.


Logs from `cat ~/Library/Logs/Zed/Zed.log`
```
{
  "scope": "local",
  "uri": "file:////PaymentMethodsList.tsx",
  "libraryPath": "/Users/uladzislauramanovich/Development/brand-app/node_modules/eslint/lib/api.js"
}
2024-02-10T09:12:18+01:00 [INFO] 18 unhandled notification eslint/confirmESLintExecution:
{
  "scope": "local",
  "uri": "file:///Player.tsx",
  "libraryPath": "/Users/uladzislauramanovich/Development/brand-app/node_modules/eslint/lib/api.js"
}
2024-02-10T09:12:20+01:00 [INFO] Node runtime install_if_needed
2024-02-10T09:12:22+01:00 [INFO] Node runtime install_if_needed
2024-02-10T09:12:23+01:00 [WARN] timed out waiting for formatting
2024-02-10T09:13:17+01:00 [ERROR] crates/gpui/src/window.rs:404: window not found
2024-02-10T09:13:17+01:00 [ERROR] crates/lsp/src/lsp.rs:624: oneshot canceled
2024-02-10T09:13:17+01:00 [ERROR] crates/lsp/src/lsp.rs:624: oneshot canceled
2024-02-10T09:13:17+01:00 [ERROR] crates/lsp/src/lsp.rs:624: oneshot canceled
2024-02-10T09:13:17+01:00 [ERROR] crates/lsp/src/lsp.rs:624: oneshot canceled
2024-02-10T09:15:06+01:00 [INFO] build git repository ".git"
2024-02-10T09:15:06+01:00 [INFO] starting language server "typescript-language-server", path: "/Users/uladzislauramanovich/Development/brand-app", id: 21
2024-02-10T09:15:06+01:00 [INFO] starting language server "eslint", path: "/Users/uladzislauramanovich/Development/brand-app", id: 22
2024-02-10T09:15:06+01:00 [INFO] 22 unhandled notification window/logMessage:
{
  "type": 3,
  "message": "ESLint server running in node v18.15.0"
}
2024-02-10T09:15:06+01:00 [INFO] 21 unhandled notification window/logMessage:
{
  "type": 3,
  "message": "Using Typescript version (user-setting) 5.3.2 from path \"/Users/uladzislauramanovich/Development/brand-app/node_modules/typescript/lib/tsserver.js\""
}
2024-02-10T09:15:06+01:00 [INFO] 21 unhandled notification $/typescriptVersion:
{
  "version": "5.3.2",
  "source": "user-setting"
}
2024-02-10T09:15:06+01:00 [INFO] 22 unhandled notification eslint/confirmESLintExecution:
{
  "scope": "local",
  "uri": "file:///Users/uladzislauramanovich/Development/brand-app/api/storyblok.ts",
  "libraryPath": "/Users/uladzislauramanovich/Development/brand-app/node_modules/eslint/lib/api.js"
}
2024-02-10T09:15:06+01:00 [INFO] 22 unhandled notification eslint/confirmESLintExecution:
{
  "scope": "local",
  "uri": "file:///Users/uladzislauramanovich/Development/brand-app/types/api/storyblok/paymentMethods.ts",
  "libraryPath": "/Users/uladzislauramanovich/Development/brand-app/node_modules/eslint/lib/api.js"
}
```

--- a/crates/zed/src/languages/typescript.rs
+++ b/crates/zed/src/languages/typescript.rs
@@ -259,7 +259,7 @@ impl LspAdapter for EsLintLspAdapter {
         let release = latest_github_release(
             "microsoft/vscode-eslint",
             false,
-            false,
+            true,
             delegate.http_client(),
         )
         .await?;

